src/main/java/core/basesyntax/strategy/impl/OperationSupply.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class OperationSupply implements OperationHandler {
+    private static final FruitStorageDao storageDao = new FruitStorageDaoImpl();
```

Comment:
Let's create only one instance of your fruitDao class and pass it to the constructor where needed, your services must be reusable.